pywbem/_subscription_manager.py

```diff
@@ -207,6 +206,7 @@ def __init__(self, subscription_manager_id):
         self._owned_subscriptions = {}  # CIMInstance of owned subscriptions
         self._owned_filters = {}  # CIMInstance of owned filters
         self._owned_destinations = {}  # CIMInstance owned destinations
+        self._systemnames = {}  # Dict that will contain SystemNames
```

Comment:
The new attribute should be added to `__str__()` and `__repr__()`.

Reply:
DONE